src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Values/HostingEnvironmentValue.cs

```diff
@@ -0,0 +1,69 @@
+using System.Globalization;
+using System.Text.Encodings.Web;
+using Fluid;
+using Fluid.Values;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OrchardCore.Liquid;
+
+namespace OrchardCore.DisplayManagement.Liquid.Values;
+
+internal sealed class HostingEnvironmentValue : FluidValue
```

Comment:
What's the advantage of this implementation over the current one?

Reply:
I see, thanks. @hishamco please document if anything changes here: https://docs.orchardcore.net/en/latest/reference/modules/Liquid/#environment

Reply:
Nothing to be changed in the usage, no breaking-changes